Content.Shared/Buckle/SharedBuckleSystem.Buckle.cs

```diff
@@ -436,9 +436,11 @@ public bool TryUnbuckle(EntityUid buckleUid, EntityUid userUid, bool force = fal
                 return false;
 
             // If the strap is a vehicle and the rider is not the person unbuckling, return.
-            if (TryComp<VehicleComponent>(strapUid, out var vehicle) &&
-                vehicle.Rider != userUid)
-                return false;
+            if (TryComp<VehicleComponent>(strapUid, out var vehicle) && vehicle.Rider != userUid)
```

Comment:
use braces for multi-line if statements

Reply:
is this needed if i do the right up next requested change?